site/styles.css

```diff
@@ -15,14 +15,19 @@ h2 {
 .header-box {
     display: grid;
     color: lightgray;
-    padding: 350px;
     background-color: gray;
+
+}
+
+.header-box-image {
+    object-fit: fill;
```

Comment:
Nice! Let me know how this is working @itisjamied. I'm not 100% sure but you may need some more styles on this for `object-fit` to have an effect, I could be wrong though.